content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,181 @@ aliases:
+We are working on the upgrade process for users running Docker and 1.8.x.
+Hold off on upgrading for now.
 {{% /note %}}
 
-1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/),
-   unpackage the InfluxDB binaries, and then place them in your `$PATH`.   
+There is no plan for a 32-bit build of InfluxDB 2.0.
+
+### Continuous queries
+
+If you use continuous queries, review them prior to upgrading by running the `show continuous queries` command:
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save your continuous queries output to use later.
+
+Continuous queries are replaced by tasks in InfluxDB 2.0
+To convert your continuous queries to Flux, see [Get started with InfluxDB tasks](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+InfluxDB 2.0 doesn't support these protocols directly.
+Use [Telegraf](/telegraf/v1.16/) to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+- Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+  Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+- Kapacitor Stream-style TICKscripts will not work using the [subscription API](/influxdb/v1.8/administration/subscription-management/).
+  (There is no subscriptions API in InfluxDB 2.0.)
+  We recommend writing data directly to *both* Kapacitor and InfluxDB to allow stream tasks to continue to work.
+  To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+  If you use other data sources to send data to InfluxDB, Telegraf can be used as an intermediate layer to write to both Kapacitor and InfluxDB.
+  <!-- 
+  Other architectures and mechanisms are possible, too.
+  Need more help? Reach out in the [community forums](https://community.influxdata.com).
+  -->
+
+### User migration
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, it *does not migrate administrative users*.
+
+To review users with admin permissions, in the InfluxDB 1.x CLI, run `show users`.
+Any users labeled "admin" *will not* be migrated.
+
+If you're using an admin user for visualization or Chronograf's administrative functions, you should create a new read-only user before upgrading.
+Admin rights are granted to the primary user created in the InfluxDB 2.0 setup process which runs at the end of the upgrade process.
+(This provides you with the opportunity to re-assess who should be granted admin-level access in your InfluxDB 2.0 setup.)
```

Comment:
If you're using an admin user for visualization or Chronograf's administrative functions, ~you should~ create a new read-only user before upgrading.
Admin rights are granted to the primary user created in the InfluxDB 2.0 setup process which runs at the end of the upgrade process.
This provides you with the opportunity to re-assess who should be granted admin-level access in your InfluxDB 2.0 setup.